scripts/deploy.sh

```diff
@@ -7,17 +7,88 @@ pull_latest() {
 
 rebuild_regs() {
   # Build WCA regulations
-  # Use only one core because we get better backtraces without multiprocessing,
-  # and because we might be on the production server and should not eat up all
-  # cores just building the regulations.
-  regulations/make.py --num-workers 1 --wca
-  regs_folder=WcaOnRails/app/views/regulations
+  # Uses wrc, see here: https://github.com/cubing/wca-regulations-compiler
```

Comment:
What exactly would you want to move there?

I had in mind that the website would say "i want html for this, i want pdf for that", which is the case currently, but maybe you pictured something more like "i want all the views for the regulations"?